app/models/cart.rb

```diff
@@ -5,9 +5,9 @@ def initialize(contents)
     @contents = contents
   end
 
-  def add_item(item)
```

Comment:
I like this passing the Id in instead of the whole object should speed the program up

Reply:
Totally agree here, no reason to pass an entire object when all you really need is an id.

Reply:
i just edited the name.  i thought it was the whole item originally as well, and spent quite some time trying to get it to work, only to realize they just named it wrong